examples/service-mesh/istio.json

```diff
@@ -0,0 +1,43 @@
+{
+    "apiVersion": "vlabs",
+    "properties": {
+      "orchestratorProfile": {
+        "orchestratorType": "Kubernetes",
+        "orchestratorRelease": "1.9",
+        "kubernetesConfig": {
+          "apiServerConfig": {
+            "--admission-control":  "NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DenyEscalatingExec,Initializers,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota",
+            "--runtime-config": "admissionregistration.k8s.io/v1alpha1"
+          }
+        } 
+      },
+      "masterProfile": {
+        "count": 1,
+        "dnsPrefix": "acseistio",

```

Comment:
The dnsPrefix should be empty to be consistent with other examples/ apimodels (also so we can inject it in our e2e test)